recipes/clad/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "clad" %}
+{% set version = "v0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vgvassilev/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: f93f02c1c03c8daf54c34ec81a46ad13f7431b95a1809e96d26126cb8d0233cc
+  folder: source
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make  # [unix]
+    - ninja  # [win]
+    - {{ compiler('cxx') }}
+  host:
+    - llvm
+    - llvmdev
+    - clang
+    - clangdev
+
+test:
+  commands:
+    - test -f $PREFIX/lib/clad.so  # [unix]
+    - if not exist %LIBRARY_BIN%\\clad.dll exit 1  # [win]
+
+about:
+  home: https://github.com/vgvassilev/clad
+  license: LGPL-3.0
+  license_family: LGPL
+  license_file: License.txt
+  summary: clad -- automatic differentiation for C/C++
+  description: |
+    Clad enables automatic differentiation (AD) for C++. It is based on LLVM compiler infrastructure and is a plugin for Clang compiler.
+  doc_url: https://github.com/vgvassilev/clad/blob/master/README.md
+  dev_url: https://github.com/vgvassilev/clad/wiki
+
+extra:
+  recipe-maintainers:
+    - vgvassilev
```

Comment:
@vgvassilev can you please confirm that you are in a list of maintainers?

Reply:
I confirm.